Yoast/wordpress-seo
# Making "Analysis results" translatable and removing the colon

## Summary

This PR can be summarized in the following changelog entry:

* N/A

## Relevant technical choices:

* N/A

## Test instructions

This PR can be tested by following these steps:

* checkout this branch in your plugin folder and build this pr by executing `yarn&&composer install&&grunt`
* Verify The colon is gone like in the picture below:
<img width="248" alt="cat___ _qwerty_ _wordpress" src="https://user-images.githubusercontent.com/5509155/44031492-78a54e2e-9f04-11e8-9203-a2562a63a8fa.png">


## Quality assurance


Fixes a part of #10596
